cmd/initcmd/cmdinit.go

```diff
@@ -24,6 +24,6 @@ func NewCmdInit(ioStreams genericclioptions.IOStreams) *cobra.Command {
 			cmdutil.CheckErr(io.Run())
 		},
 	}
-	cmd.Flags().StringVarP(&io.GroupName, "group-name", "g", "", "Name to group applied resources. Must be composed of valid label characters.")
+	cmd.Flags().StringVarP(&io.InventoryName, "inventory-name", "i", "", "Name to group applied resources. Must be composed of valid label characters.")
```

Comment:
I think we should consider calling the value of the label the `prune inventory id` rather than use `name`. It is the identifier we use to link from the template manifest to the actual objects. And if we move towards setting the value simply to a random generated value, using `name` doesn't quite feel right. Also, it removes the ambiguity between the value in the label and the name of the actual object.

Reply:
I have taken your suggestion and changed `InventoryName` to `InventoryID`. Please note that the linter forces both letters of `ID` to be capitalized. I've also changed the flag from `inventory-namel` to `inventory-id`. I believe you are right that `name` does not make sense.

However, I will stick with the `Inventory` instead of the longer `PruneInventory`. I think this is in keeping with golang's shorter identifiers, and Brian Grant has already blessed it.